domains/tuanpham.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Tun52299",
+           "email": "",
+           "discord": "699291160157618226",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.W4jaWyOg563nkn4s5Cat0abWtQHjDMiS-Cwk_g-hZTTSTGkGhZcj-cT-Wwfhf85xwKm9cY98ub9LIme6RI-WEL9zkS6Uogr8D2stgI7GvckH3K6YniTHXD8ZA_1Xw_mAVIxYG0nPMs_ZJMeTBOlvGRHpAk7AUXBGjg7pw3Izf4Heolw5Cvje2rPyUY7fXb7hl7mp2rx5VBWS0t5m1H00V7-u5w7xlvVWeF399lzWUusI3st0N8nyrHiQwQPCL9Up_vQAG00axehhbvVbUx99iR6wcrB0B4sRfnsaPOLMPa_XwkVGUoH3trk5BdtztfCWev_ehv8VwARb9yFNinWGMw.Hiq8mUttqfT_lJs9eJV0oQ.IIocQ3pPOAB-DMQiP7Qal06D5fs2OnBvh9YIFUCvih3rbAjOLRRmHq6nFzH-zibVgrEQOVQNVnhc4FJPVPfF7IAmW_64hcCQLP3QOy5woTc.wNfoqwymGPil8HOqliP5Xg"
+        },
+    
+        "record": {
+            "CNAME": "tunraena.github.io"
```

Comment:
CNAME returns a 404, can you send a site preview?

Reply:
Is this the preview you need   https://github.com/Tun52299/Tun52299.github.io